tests/topotests/lib/common_config.py

```diff
@@ -4985,3 +4985,52 @@ def scapy_send_raw_packet(tgen, topo, senderRouter, intf, packet=None):
 
     logger.debug("Exiting lib API: {}".format(sys._getframe().f_code.co_name))
     return True
+
+
+def kernel_requires_l3mdev_adjustment():
```

Comment:
@donaldsharp , This function is not being added in this PR, it is already present, written by someone else from community.  It is showing up only because of some white space addition.